libs/expo/betterangels/src/lib/screens/AddEditClient/index.tsx

```diff
@@ -128,39 +129,40 @@ export default function AddEditClient({ id }: { id?: string }) {
   const onSubmit: SubmitHandler<
     UpdateClientProfileInput | CreateClientProfileInput
   > = async (values) => {
-    if (values.contacts && values.contacts?.length > 0) {
-      values.contacts = values.contacts.map((contact) => ({
-        ...contact,
-        phoneNumber: contact.phoneNumber === '' ? null : contact.phoneNumber,
-      }));
-    }
-
-    const filteredSocialMediaProfiles =
-      values.socialMediaProfiles?.filter((item) => item.platformUserId) || [];
+    if (isCreating || isUpdating) return;
```

Comment:
I think you can remove this. Adding `isCreating || isUpdating` to `isLoading`, on line 406, should achieve the desired result